lib/rex/post/meterpreter/extensions/stdapi/mic/mic.rb

```diff
@@ -0,0 +1,61 @@
+# -*- coding: binary -*-
+
+require 'rex/post/meterpreter/channel'
+require 'rex/post/meterpreter/channels/pools/audio_stream_pool'
+
+module Rex
+  module Post
+    module Meterpreter
+      module Extensions
+        module Stdapi
+          module Mic
+
+            ###
+            #
+            # This meterpreter extension can list and capture from microphone
+            #
+            ###
+            class Mic
+              include Msf::Post::Common
```

Comment:
`Rex` is not supposed to depend on anything in the `Msf` namespace. I don't see anything here that uses this, can you remove it?